eagle-external/hadoop_jmx_collector/hadoop_jmx_kafka.py

```diff
@@ -64,6 +64,23 @@ def on_bean(self, bean):
             capacityusage = round(float(bean['CapacityUsed']) / float(bean['CapacityTotal']) * 100, 2)
             self.collector.on_bean_kv(self.PREFIX, "capacityusage", capacityusage)
 
+class NNNodeUsageMetric(JmxMetricListener):
+    PREFIX = "hadoop.namenode.nodeusage"
+
+    def on_bean(self, bean):
+        if bean["name"] == "Hadoop:service=NameNode,name=NameNodeInfo":
+            nodeusagedic = json.loads(bean["NodeUsage"]);
+            nodeusage_detail_dic = nodeusagedic["nodeUsage"];
+            min = round( float(nodeusage_detail_dic["min"].strip('%')) ,2 );
```

Comment:
In order to avoid confusing, could you pls. rename local variable min/max to min_val/max_val as min and max are Python <built-in function> names?